akka-http2-support/src/main/scala/akka/http/impl/engine/http2/Http2Multiplexer.scala

```diff
@@ -111,6 +111,7 @@ private[http2] trait Http2MultiplexerSupport { logic: GraphStageLogic with Stage
           buffer = ByteString.empty
           trailer = None
           maybeInlet.foreach(_.cancel())
+          state.closeStream(this)
```

Comment:
Maybe also add a forwarder in the parent class to avoid accessing `state` from inside here. (Not a real problem just to make it consistent with the other code).